code/game/machinery/stasis.dm

```diff
@@ -158,12 +158,13 @@
 	. = ..()
 	return default_deconstruction_crowbar(I) || .
 
-/obj/machinery/stasis/nap_violation(mob/violator)
-	unbuckle_mob(violator, TRUE)
-
+//for some reason, in testing, cyborgs were unable to unbuckle people from stasis beds unless this extension of attack_robot() here was present, so I guess it stays in
```

Comment:
The reason is probably just because https://github.com/tgstation/tgstation/blob/305cebb2c09fd957cfdbbc2f533108ec4c9f1b2a/code/game/machinery/_machinery.dm#L322 doesn't call it's parent.
It would be preferable to find a more general solution so cyborgs can unbuckle from all kinds of machinery (like tesla coils and pipes).